.gitignore

```diff
@@ -1,3 +1,4 @@
 *.kdev4
 build/
 src/version.h
+/example/*.fits
```

Comment:
A worthwhile thing to have in the gitignore, but should it really be part of this PR?

Reply:
Not sure what the other think about this, but I don't personally find a single line in the `.gitignore` to be worth a PR on its own :sweat_smile:  Also, we can't push directly to the branch, so I made it sneak into this PR :grin:

Reply:
I factored it out in a commit on its own, at least :slightly_smiling_face: